ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/Services.py

```diff
@@ -103,3 +105,17 @@ def omit_service(model_list, ignore_list, service_name, log):
 
     # model all services by default
     return False
+
+
+def truncate_service_class(group_list, service_name, service_caption):
+    service_class_name = service_name
+    service_class_caption = service_caption
+    if group_list:
+        for group in group_list:
+            if re.search(group, service_name):
+                service_class_name = re.search(group, service_name).group(0)
+                service_class_caption = re.sub("(?:.(?!_.*))+$", '', service_caption)
+                return service_class_name, service_class_caption
+        return service_class_name, service_class_caption
+    else:
+        return service_class_name, service_class_caption
```

Comment:
I  don't think you need the 'else' section.  Just set the default to return at the end.